nixos/modules/services/monitoring/prometheus/exporters/rspamd.nix

```diff
@@ -0,0 +1,99 @@
+{ config, lib, pkgs }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.rspamd;
+
+  prettyJSON = conf:
```

Comment:
Possibly use the same name here (writePrettyJSON) as in prometheus/default.nix, making it immediately clear why this is done.